### PR TITLE
15yr button component

### PR DESCRIPTION
**Pull request by ryan-ludwig**
Component to keep buttons consistent on 15yr page and global banner.

### CODE REVIEW

File: src/components/15Years/15YearsButton.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	computed: {
+		tag() {
+			if (this.to) {
+				return 'router-link';
+			}
+			return this.href ? 'a' : 'button';
+		}
+	},
+	methods: {
+		onClick(e) {
```
Comment (ryan-ludwig) [inline src/components/15Years/15YearsButton.vue:45]: Testing this out. If it seems solid we can add to KvButton so in templates we can just write `<kv-button @click="myMethod">` and avoid the pitfall of forgetting `<kv-button @click.native="myMethod">`

Comment (ryan-ludwig) [reply src/components/15Years/15YearsButton.vue:45]: VUE-354

Comment (emuvente) [reply src/components/15Years/15YearsButton.vue:45]: 👍  great improvement!

**Review by emuvente — APPROVED** [review]

File: src/components/15Years/15YearsButton.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
+<script>
+export default {
+	render(createElement) {
+		const options = {
+			class: ['fifteen-yr-button', this.variant ? `fifteen-yr-button--${this.variant}` : ''],
```
Comment (ryan-ludwig) [inline src/components/15Years/15YearsButton.vue:5]: This component is a duplicate of KvButton but with a 'variant' prop and scoped styles. And a click handler override.

**Pull request merged**